test/mocks/mockMiniMemos.js

```diff
@@ -37,13 +39,14 @@ const mockedMiniMemosPdfAndWeb = {
         memoFlag: 'historyMemo',
         pdfPMDate: '2019-09-05',
         pdfMemoUploadDate: '2021-07-07T14:35:31.507Z',
-        lastChangeDate: '2021-07-07T14:35:31.507Z',
-      },
+        lastChangeDate: '2021-07-07T14:35:31.507Z'
+      }
     },
     20192: {
       1: {
         _id: 'EI1220_20192_1',
         courseCode: 'EI1220',
+        applicationCodes: ['1'],
         changedBy: 'zzzz',
         ladokRoundIds: ['1'],
```

Comment:
`ladokRoundIds ` and `roundIdList` has to be removed everywhere, in order to be sure tests work

Reply:
ladokRoundIds and roundIdList was removed from everywhere